akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedRunning.scala

```diff
@@ -38,10 +36,12 @@ import akka.actor.typed.internal.PoisonPill
  * This is implemented as such to avoid creating many EventsourcedRunning instances,
  * which perform the Persistence extension lookup on creation and similar things (config lookup)
  *
- * See previous [[EventsourcedReplayingEvents]].
+ * See previous [[ReplayingEvents]].
+ * TODO rename
  */
 @InternalApi
 private[akka] object EventsourcedRunning {
```

Comment:
We can't name it just `Running`, any suggestions? Keep as it is?

Reply:
I don't mind `Running`, or `HandlingCommands`